content/en/tracing/profiling/java.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Profiling Java Applications
+kind: documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+The Datadog Profiling library supports the Java JRE 11 and higher of both Oracle JDK and OpenJDK. Early-access versions of Java aren't supported.
+Java profiling and tracing are shipped within the same `dd-java-agent.jar` library. In order to use tracing or profiling, you must enable these functions. To enable tracing, follow the [enable tracing documentation][1].
+
+**Note**: Some specific builds will work with Datadog profiling if they support [Java Flight Recorder][2].
+
+## Getting Started
+
+To begin profiling applications written in any JVM based runtime:
+
+1. Download `dd-java-agent.jar`, which contains the Agent class files:
+
+    ```shell
+    wget -O dd-java-agent.jar 'https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.datadoghq&a=dd-java-agent&v=LATEST'
+    ```
+
+2. Add the following JVM argument when starting your application in your IDE, Maven or Gradle application script, or `java -jar` command:
+
+    ```shell
+    -javaagent:<PATH>/dd-java-agent.jar
+    ```
+
+3. Enable profiling by adding `-Ddd.profiling.enabled=true` to the JVM command line (in addition to any existing APM Agent parameters).
```

Comment:
or env var, right?

Reply:
indeed.

Reply:
99% sure. @mar-kolya can you san-check please?

Reply:
`-Ddd.profiling.enabled=true` - correct.